### PR TITLE
Updated Envoy documentation to include new variables

### PR DESCRIPTION
**Pull request by gtdiehl**
**Description:**
Documented new optional configuration variables for the Enphase Envoy sensor

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#28837

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
Change that requires a Home Assistant release (depending on a PR in the main code base) needs to be based off and go into the `next` branch.

Please update the target branch of this PR to `next` and rebase the PR if needed.

**Comment by gtdiehl** [comment]
@frenck I can change the branch I'm going to commit into from `current` to `next`, but my branch where my changes are at stays at `current`. If I do change the branch alot of commits besides mine are listed as going to be commited

Do I need to close this PR and open a new one?

**Comment by frenck** [comment]
No, you don't need to create a new one. You need to change the target branch in GitHub. At that point, a lot of unrelated commits show up. You next have to rebase your PR (your commits) onto the `next` branch using the git command line.

**Comment by frenck** [comment]
@gtdiehl Small and friendly notification. Are you going to pick this up?
We can't merge this way and this will block the parent PR.

**Comment by gtdiehl** [comment]
@frenck Sorry, I've been busy lately.  I'll fix this today as I dont want to hold up anything.

**Comment by gtdiehl** [comment]
@frenck I changed the target branch to `next`.  Again sorry for the delay.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request merged**
